pydrive2/fs/spec.py

```diff
@@ -49,13 +55,118 @@ def should_retry(exc):
     )(func)
 
 
+def _cache_file(client_id, client_user_email=None):
+    cache_dir = appdirs.user_cache_dir("pydrive2fs", appauthor=False)
+    client_cache_dir = os.path.join(cache_dir, client_id)
+    name = client_user_email or "default"
```

Comment:
Probably need to escape some stuff in client_id and email to make it a legal filename.

Reply:
Though it seems kinda rare, so probably not worth bothering with until there is need.